main.go

```diff
@@ -117,14 +132,15 @@ func main() {
 		}
 
 		if healthy {
-			if silenceReq.Active() {
-				// TODO: There might be more than one silence to remove
-				err = silenceReq.Remove()
-				if err != nil {
-					log.Fatal(err)
-				}
-			} else {
-				log.Println("Health checks passed and cluster was not silenced. Nothing to do here.")
+			// Use this to log how much time was left in the active silence. This can
```

Comment:
In a run I just finished on a stable and otherwise-idle stage cluster, this was 6m31.917376129s.

Reply:
This would be an extremely interesting metric that we could add to prometheus and export via the osd-metrics-exporter

Reply:
Agree. However, if we take the approach described in OSD-6646 that'll be moot.